lucetc/src/error.rs

```diff
@@ -9,31 +9,31 @@ use thiserror::Error;
 pub enum Error {
     //
     // General #[from] implementations.
-    #[error("Builtins error")]
+    #[error("Builtins: {0}")]
     Builtins(#[from] parity_wasm::elements::Error),
-    #[error("Clif module error")]
+    #[error("Clif module: {0}")]
     ClifModuleError(#[from] ClifModuleError),
-    #[error("Translating error")]
+    #[error("Translating: {0}")]
     ClifWasmError(#[from] ClifWasmError),
-    #[error("Lucet Module error")]
+    #[error("Lucet Module: {0}")]
```

Comment:
this is definitely a "when we actually spec this out with tests" thought, but "Error: Lucet Module error: Unknown module for ..." ends up repeating that there's an error a bit more than we probably should :sweat_smile: